llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -659,7 +659,7 @@ def Dot3 :  DXILOp<55, dot3> {
 
 def Dot4 :  DXILOp<56, dot4> {
   let Doc = "dot product of two float vectors Dot(a,b) = a[0]*b[0] + ... + "
-            "a[n]*b[n] where n is between 0 and 3";
+            "a[n]*b[n] where n is 0 to 3 inclusive";
```

Comment:
Just something incidental as I found these descriptions misleading since the only numbers "between" 0 and 3 are 1 and 2.